commons-vfs2/src/test/java/org/apache/commons/vfs2/provider/sftp/SftpPermissionExceptionTestCase.java

```diff
@@ -105,7 +105,7 @@ public void testGetOutputStreamException() throws Exception {
         }
 
         // try to get created channel number.
-        final int channelId = server.getActiveSessions().get(0).registerChannel(new ChannelSession());
+        final long channelId = server.getActiveSessions().get(0).getIoSession().getId();
```

Comment:
(not sure if this makes any sense, to be looked at in more detail when the change seems otherwise useful)